Apps/WebClient/src/ClientApp/src/store/modules/vaccinationStatus/actions.ts

```diff
@@ -52,6 +53,35 @@ export const actions: VaccinationStatusActions = {
                 });
         });
     },
+    getReport(
+        context,
+        params: { phn: string; dateOfBirth: StringISODate }
+    ): Promise<Report> {
```

Comment:
This should also be added to the VaccinationStatusActions interface for consistency.